packages/base/src/Internal/Algorithms.hs

```diff
@@ -170,7 +170,8 @@ a = (5><3)
 -0.690  -0.352   0.433  -0.233   0.398
 
 >>> s
-fromList [35.18264833189422,1.4769076999800903,1.089145439970417e-15]
+[35.18264833189422,1.4769076999800903,1.089145439970417e-15]
+it :: Vector Double
```

Comment:
Why the "it"?

Reply:
I used the ":set +t" option in the ghci session while replaying the haddock examples,
initially just to help myself working out the problems, but than I thought better.
With the old Show instance the fromList  prefix indicated that, for example, [1,2] is not
a list but something else. With the new vector Show instance that was not
visually indicated. I thought that including the type of the expression after each
evaluation could be useful for the reader, in particular when the type is not
immediately evident for the casually browsing reader.

Reply:
👍